src/graphql.rs

```diff
@@ -135,7 +136,10 @@ impl fmt::Display for GraphQLError {
             ),
             GraphQLError::ServerError => write!(f, "There was an error on our server handling your request.\nOur ops team has been alerted and is investigating the issue."),
             GraphQLError::UnauthorizedError(resource, operation) => write!(f, "The access token is not authorized to {} this {}.", operation, resource),
-            GraphQLError::ValidationError(field, message) => write!(f, "There was a problem with a value you supplied: {} {}.", field, message)
+            GraphQLError::ValidationError(field, message) => write!(f, "There was a problem with a value you supplied: {} {}.", field, message),
+            GraphQLError::ProjectNotFoundError(name) => {
+                write!(f, "Unable to find project '{}'", name)
```

Comment:
The other error messages use punctuation. We can drop that if you prefer, but the messages should be consistent with each other.

Reply:
Good catch.